playbooks/simple/create_role_assignment.yaml

```diff
@@ -0,0 +1,30 @@
+---
+- name: Create API client and Role Assignment
+  hosts: localhost
+  tasks:
+    -
+#    - name: Create a Private RSA key
+#      community.crypto.openssl_privatekey:
+#        path: "priv_rsa.pem"
+#        type: RSA
+#        size: "2048"
+#
+#    - name: Create a Public RSA key
+#      community.crypto.openssl_publickey:
+#        path: "pub_rsa.pem"
+#        privatekey_path: "priv_rsa.pem"
+#
+#    - name: Create new API client foo
+#      purestorage.fusion.fusion_api_client:
+#        issuer_id: "{{ ansible_env.FUSION_ISSUER_ID }}"
+#        key_file: "{{ ansible_env.FUSION_PRIVATE_KEY_FILE }}"
+#        state: present
+#        name: "test_api_client"
+#        public_key: "{{ lookup('file', 'pub_rsa.pem') }}"
```

Comment:
Why this commented lines?

Reply:
whoops, I created `[create_role_assignment.yaml](https://github.com/Pure-Storage-Ansible/Fusion-Collection/pull/135/files#diff-df40034ca1f34f237c3f4055e887a546d99282c4c15277367c7a949a8d0a0c87)` just so I can test the changes. Should I delete the file or uncomment the tasks and leave it there together with other simple playbooks?

Reply:
It's up to you. If there's no reason why you added the playbook, you can just delete it.